pulsar/json_schemas/channel.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://asyncapi.com/bindings/pulsar/channel.json",
+    "title": "Channel Schema",
+    "description": "This object contains information about the channel representation in Pulsar.",
```

Comment:
Let's define that this `channel` defines Pulsar namespace/topic level admin configs here in the description to clarify.

Reply:
Done